flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/ATSU/A320_Neo_CDU_AOC_AirportList.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * This method assembles the airport data which contains the raw data as well as the OutPut that can be seen on the page
+ * @param _icao {string}
+ * @param _isManaged {boolean}
+ * @returns {{outPut: string, icao: string, isManaged: boolean}}
+ */
+function assembleAirportInfo(_icao = "", _isManaged = false) {
+    return { icao: _icao, isManaged: _isManaged, outPut: `{${!_isManaged || !_icao ? "cyan" : "green"}}${_icao ? _icao : "[ ]"}{end}` };
```

Comment:
Can we please use `output` instead of `outPut`? :sweat_smile:  I've never seen the latter.

Reply:
WILCO